packages/blockchain-extension/test/data/1 Org Local Fabric/start.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Copyright IBM Corp All Rights Reserved
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Exit on first error, print all commands.
+set -ev
+CONTAINER=$(docker ps -f label=fabric-environment-name="1 Org Local Fabric" -q -a)
+if [ -z "$CONTAINER" ]
+then
+    export MICROFAB_CONFIG='{"port":8080, "endorsing_organizations": [{"name": "Org1"}],"channels": [{"name": "mychannel","endorsing_organizations": ["Org1"]}]}'
+    docker run -e MICROFAB_CONFIG --label fabric-environment-name="1 Org Local Fabric" -d -p 8080:8080 ibmcom/ibp-microfab:0.0.6
+else
+    for CONTAINER in $(docker ps -f label=fabric-environment-name="1 Org Local Fabric" -q -a); do
+        docker start ${CONTAINER}
+    done
```

Comment:
I believed the `fabric-environment-name` label was unique. If that is not the case ignore this comment. If it is, then we don't need a for loop, only `docker start ${CONTAINER}`.